Build/scripts/post_build.ps1

```diff
@@ -34,6 +34,8 @@ param (
     [string[]]$pogo = @(),
     [string]$pogoscript = "",
 
+    [string]$buildTypeSuffix = "",
```

Comment:
Yeah, to support build configs where the output folder gets the _pogo suffix and the script needs to know about that directory specifically.